code/game/objects/items/devices/glue.dm

```diff
@@ -16,19 +17,36 @@
 	if(!proximity || !target)
 		return
 	else
-		if(uses == 0)
-			to_chat(user, "<span class='warning'>The bottle of glue is empty!</span>")
+		if(uses <= 0) //Just in case it somehow goes below 0
+			to_chat(user, "<span class='warning'>\The [name] is empty!</span>")
 			return
 		if(istype(target, /obj/item))
 			var/obj/item/I = target
 			if(HAS_TRAIT_FROM(I, TRAIT_NODROP, GLUED_ITEM_TRAIT))
-				to_chat(user, "<span class='warning'>[I] is already sticky!</span>")
+				to_chat(user, "<span class='warning'>\The [I] is already sticky!</span>")
 				return
 			uses -= 1
-			ADD_TRAIT(I, TRAIT_NODROP, GLUED_ITEM_TRAIT)
-			I.desc += " It looks sticky."
-			to_chat(user, "<span class='notice'>You smear the [I] with glue, making it incredibly sticky!</span>")
-			if(uses == 0)
+			if(uses <= 0)
 				icon_state = "glue_used"
 				name = "empty bottle of super glue"
+			if(drying_time > 0)
+				addtimer(CALLBACK(I, PROC_REF(get_glued)), drying_time)
+				I.register_glue_signals()
+				to_chat(user, "<span class='notice'>You smear \the [I] with glue, which will make it incredibly sticky once the glue dries!</span>")
+			else
+				get_glued()
+				to_chat(user, "<span class='notice'>You smear \the [I] with glue, making it incredibly sticky!</span>")
 			return
+
+/obj/item/proc/register_glue_signals()
+	RegisterSignal(src, COMSIG_ITEM_EQUIPPED, PROC_REF(get_glued))
+	RegisterSignal(src, COMSIG_ITEM_DROPPED, PROC_REF(get_glued))
```

Comment:
the purpose of this signal is a bit confusing to read. Adding a comment like `equiping or dropping a glued item will instantly dry` would be helpful.